src/components/menu/__internal__/submenu/submenu.style.js

```diff
@@ -57,6 +57,8 @@ const StyledSubmenu = styled.ul`
       }
     `}
 
+    border-bottom-left-radius: 6px;
```

Comment:
Same comment about having tokens for these (not a problem for this PR)

Reply:
Probably as it is not documented in DS documentation as per screenshot below.
<img width="881" alt="Screenshot 2023-03-30 at 13 59 49" src="https://user-images.githubusercontent.com/48626342/228843910-a7ff1346-f797-4790-bb53-bb913d0b667a.png">

Reply:
I have informed DS to add the value to the official docs.